### DISCUSSION TITLE
help installing to my server

### DISCUSSION THREAD

**Discussion by powdy** [tltneon/lgsl#41] [Q&A] [answered]
Hi, is there anyone that can help me install this to my webserver? Thanks

**Comment by tltneon** [comment, marked as answer]
Hi @powdy!
At first, you can read this article: https://github.com/tltneon/lgsl/wiki/How-to-install-LGSL
If you still in trouble with installing LGSL, write my what exactly you did.

**Reply by powdy** [reply]
Hi, ive tried a few times to install it, i have read though it but the first part has got me stuck. I can create mysql and unzip to my public_html etc. I searched through the php.ini but cant find the parts i need to uncomment. Is there anyway i could contact you through DM? Thanks for the reply

**Reply by tltneon** [reply]
Sorry I don't know that is the DM :)
LGSL will work even if php.ini lines still uncommented, it's not a important thing to installing (only for quering a servers).
1. Unzip to **public_html** (rename lgsl-master to lgsl)
2. Make **lgsl** database in mysql
3. Go to **yourwebsite.com/lgsl/install.php** and proceed the installation script
3.1. Step 1: fill the inputs: db login and password and press **"Create tables"**
3.2. Step 2: set up setting as you want, generate config and paste it into **/public_html/lgsl/lgsl_files/lgsl_config.php**
